### ISSUE TITLE
Visual Studio Crashes On Hard Coded Text Value Suggestion

### ISSUE THREAD

**Issue by scottkuhl** [mrlacey/Rapid-XAML-Toolkit#370]
**Describe the bug**

If you set a Text value in a Label control in Xamarin.Forms to a hard coded string you get the suggestion to correct it:  

Label contains hard-coded Text value '....'.

Clicking on the suggestion bulb will cause Visual Studio to crash.

**To Reproduce**

1. Create a new Xamarin Forms project.
2. Add a new Blank Page.
3. Set a hard coded Text value in a Label.
4. Click the suggestion.

Here is the exact XAML I used to produce the error:

```xaml
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage
    x:Class="Farnsworth.Pages.Subscribe.SubscribeView"
    xmlns="http://xamarin.com/schemas/2014/forms"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:d="http://xamarin.com/schemas/2014/forms/design"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">
    <ContentPage.Content>
        <Label
            HorizontalOptions="Center"
            Text="Subscribe View"
            VerticalOptions="Center" />
    </ContentPage.Content>
</ContentPage>
```

**Expected behavior**

It helps you correct the problem.

**Additional context**

I was able to narrow this down to the Rapid XAML Toolkit by disabling the extension, which made the suggestion go away.

Microsoft Visual Studio Community 2019 Version 16.6.3
Xamarin.Form 4.7.0.1080
Rapid XAML Analysis 0.10.1

Other XAML based extensions installed:
XAML Styler 3.2003.9

**Screenshots**

![image](https://user-images.githubusercontent.com/4465616/87270926-44b46400-c497-11ea-9b0b-4f8c627cae53.png)

![image](https://user-images.githubusercontent.com/4465616/87270945-53028000-c497-11ea-8b8a-692d63829e04.png)

**Comment by mrlacey** [comment]
I can't reproduce the crash but that it's not showing anything in the drop-down is definitely not right.
I'll add a fix for the menu not showing and hope that fixes the crashing issue for you too.

**Comment by mrlacey** [comment]
@scottkuhl I've just released version **0.10.2** to the marketplace which includes a fix that I hope will address the crash you are seeing. Please let me know if this helps.

**Comment by scottkuhl** [comment]
That has resolved the issue:

![image](https://user-images.githubusercontent.com/4465616/87743883-9a4a8280-c7af-11ea-945d-ff6e73c875ed.png)

Thanks
